libs/utils/helpers.js

```diff
@@ -19,3 +19,12 @@ export function updateLinkWithLangRoot(link) {
     return link;
   }
 }
+
+export async function customFetch({ resource, withCacheRules }) {
```

Comment:
Judging by the use cases so far, I'd suggest we default `withCacheRules` to `true` and remove the property from the method calls